ansible/roles/postgresql/defaults/main.yml

```diff
@@ -4,6 +4,7 @@ postgresql_port: 5432
 postgresql_ssl_enabled: True
 postgresql_dir_original_path: /var/lib/postgresql
 postgresql_dir_path: /opt/data/postgresql
+postgresql_backup_dir: /opt/data/backups
```

Comment:
we might want to consider putting this in `/opt/data/backups/postgres` just so the backups are more organized when we are backing up other stuff